satellites/adapters/repos/PostgresRepo.go

```diff
@@ -0,0 +1,70 @@
+package repos
+
+import (
+	"github.com/maritza05/golang-bootcamp-2020/satellites/domain"
+)
+
+type DBSatellite struct {
```

Comment:
Made a separate entity from the one in the domain, they are exactly similar with the difference that this includes tags for the DB. I didn't want it to include this information in the domain model so I created another entity specific to the database and to decouple the model from implementation details.

Reply:
Same comment as the CSV entity.